simulate-event.js

```diff
@@ -272,6 +292,9 @@ module.exports = function (element, type, options) {
     cancelable: true
   }, result(eventOptions, eventType, element, type, options), options);
 
+  // Preserve the original or use the new options if none were given.
+  if (!original) original = options;
```

Comment:
This was needed because `extend` was blowing away some of the options when passing an existing event as the `options`, most likely due to the forced `defineProperty`.